app/src/main/java/com/fmpdroid/moneybox/CreateMoneyBox.java

```diff
@@ -0,0 +1,113 @@
+
+        edtTime = findViewById(R.id.edtTime);
+        edtDate = findViewById(R.id.edtDate);
+        edtTime.setText(String.format("%02d:%02d", getCurrentHour(), getCurrentMinute()));
+        edtDate.setText(getCurrentMonthName() + " " + getCurrentDay() + ", " + getCurrentYear());
+    }
+
+    public void createMoneyBox(View v){
+
+        Toast.makeText(context, "No no no", Toast.LENGTH_SHORT).show(); //temp
+        /*
+        SharedPreferences preferences = getApplicationContext().getSharedPreferences(PREFERENCE, 0);
+        SharedPreferences.Editor editor = preferences.edit();
+        editor.putBoolean(TEMP_CHECK, false);
+        editor.commit();
+
+        Intent intent = new Intent(CreateMoneyBox.this, MainActivity.class);
+        startActivity(intent);
+        */
+    }
+
+    public void pickTime(View v){
+        TimePickerDialog timePickerDialog = new TimePickerDialog(context, new TimePickerDialog.OnTimeSetListener() {
+            @Override
+            public void onTimeSet(TimePicker timePicker, int hourOfDay, int minute) {
+                edtTime.setText(String.format("%02d:%02d", hourOfDay, minute));
+            }
+        }, getCurrentHour(), getCurrentMinute(), android.text.format.DateFormat.is24HourFormat(context));
+        timePickerDialog.show();
+    }
+
+    public void pickDate(View v){
+        DatePickerDialog datePickerDialog = new DatePickerDialog(context, new DatePickerDialog.OnDateSetListener() {
+            @Override
+            public void onDateSet(DatePicker datePicker, int year, int month, int dayOfMonth) {
+                Calendar calendar = Calendar.getInstance();
+                calendar.set(Calendar.MONTH, month); //sets the month of the calendar instance in order to retrieve its month name
+                edtDate.setText(getMonthName(calendar) + " " + dayOfMonth + ", " + year);
+            }
+        }, getCurrentYear(), getCurrentMonth(), getCurrentDay());
+        datePickerDialog.show();
+    }
+
+    @Override
+    public boolean onSupportNavigateUp() {
+        onBackPressed();
+        return true;
+    }
+
+    private int getCurrentHour(){
+        return Calendar.getInstance().get(Calendar.HOUR_OF_DAY);
+    }
+    private int getCurrentMinute(){
+        return Calendar.getInstance().get(Calendar.MINUTE);
+    }
+    private int getCurrentDay(){
+        return Calendar.getInstance().get(Calendar.DAY_OF_MONTH);
+    }
+    private int getCurrentMonth(){
+        return Calendar.getInstance().get(Calendar.MONTH);
+    }
+    private String getCurrentMonthName(){
+        return Calendar.getInstance().getDisplayName(Calendar.MONTH, Calendar.LONG, Locale.getDefault());
+    }
+    private String getMonthName(Calendar calendar){
+        return calendar.getDisplayName(Calendar.MONTH, Calendar.LONG, Locale.getDefault());
+    }
+    private int getCurrentYear(){
+        return Calendar.getInstance().get(Calendar.YEAR);
+    }
```

Comment:
I don't think we need to put these inside a method. Also, we can use LocalDate if we want to get the current date. This kind of code will look cluttered, code-wise. Moving to Utility class is better since I think we can still utilize these methods in other screens.
Laslty, merge some of these methods that are deemed necessary i.e.
getCurrentHour, getCurrentMinute, getCurrentDay and getCurrentMonth since they have the same logic - to return a value based from the given parameter.